Device Alias Support

fde31 opened the pull request:
closes #184

x37v commented:
you can't clear the name to get the default.. i wonder if that would make sense?

<img width="462" alt="image" src="https://github.com/user-attachments/assets/d3d64259-0599-4a94-86ed-9206eb42986e" />

x37v commented on src/lib/types.ts:
```diff
@@ -330,11 +337,12 @@ export type OSCQueryRNBOInstance = OSCQueryBaseNode & {
 			VALUE: undefined
-		}
+		};
+		config:OSCQueryRNBOInstanceConfig;
```
nit-pick: add space after `:`

x37v reviewed (changes requested):
defaulting display name to name is a bit problematic when there are multiple instances of the same patcher

<img width="446" alt="image" src="https://github.com/user-attachments/assets/1132ea4f-c205-4a58-b2ac-5524b4bfd322" />


<img width="460" alt="image" src="https://github.com/user-attachments/assets/f0b0119a-6cbb-4616-80fb-7605ffa78ba3" />

x37v reviewed (approved):
great!

fde31 commented:
I've added a dedicated "Reset to Default" button rather than relying on the implicit clearing of the text input and fixed the missing space.

Also my bad that it defaulted to the patcher name, rather than the jack assigned name. I've made that more clear in the code by renaming the prop and fixed the reading of the value from the state tree.

x37v reviewed (approved):
looks good!
